### PR TITLE
Restaurant Show Api Response

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: 
https://www.pivotaltracker.com/story/show/132058841

Changes proposed in this pull request:
- API for show restaurant with menus and dishes
- test for aforementioned 

What I have learned working on this feature: 
- making API:s is easy
- Testing API:s is harder than making them
- How to "get" a dynamic url for a request spec

Ready for review!

### CODE REVIEW

File: app/controllers/api/v1/restaurants_controller.rb [hunk 1]
```diff
@@ -2,4 +2,8 @@ class Api::V1::RestaurantsController < ApplicationController
   def index
     @restaurants = Restaurant.all
   end
+  def show
```
Comment (tochman) [inline app/controllers/api/v1/restaurants_controller.rb:6]: Add line brakes between methods ;-)

File: app/controllers/api/v1/restaurants_controller.rb [hunk 2]
```diff
@@ -2,4 +2,8 @@ class Api::V1::RestaurantsController < ApplicationController
...
+    @restaurant = Restaurant.find(params[:id])
+
```
Comment (tochman) [inline app/controllers/api/v1/restaurants_controller.rb]: ...but remove the if not needed inside. ;-)

File: spec/requests/api/v1/restaurants_spec.rb [hunk 3]
```diff
@@ -31,6 +32,41 @@
...
+      create(:user, email: 'whatever@random_restaurant.com', role: 'owner')
+    end
+    let!(:restaurant) { create(:restaurant, user: owner) }
+    let!(:menu) { create(:menu, restaurant: restaurant) }
+    let!(:menu2) { create(:menu, title: 'menu2', restaurant: restaurant) }
+    let!(:dish) { create(:dish, name: 'dish1', restaurant: restaurant) }
+    let!(:dish2) { create(:dish, name: 'dish2', restaurant: restaurant) }
 
+
+    it 'should return json with menus on show' do
```
Comment (tochman) [inline spec/requests/api/v1/restaurants_spec.rb]: Rephrase this test message. What are we testing here. Just the menus?

**Review by tochman — CHANGES_REQUESTED** [review]

File: spec/requests/api/v1/restaurants_spec.rb (hunk 3 above)
Comment (thesuss) [reply spec/requests/api/v1/restaurants_spec.rb]: True

File: app/controllers/api/v1/restaurants_controller.rb (hunk 1 above)
Comment (thesuss) [reply app/controllers/api/v1/restaurants_controller.rb:6]: fixed

File: app/controllers/api/v1/restaurants_controller.rb (hunk 2 above)
Comment (thesuss) [reply app/controllers/api/v1/restaurants_controller.rb]: fixed

**Review by tochman — APPROVED** [review]

**Pull request merged**